demisto_sdk/commands/common/constants.py

```diff
@@ -43,6 +44,7 @@
 LAYOUT = 'layout'
 LAYOUTS_CONTAINER = 'layoutscontainer'
 PRE_PROCESS_RULES = 'pre-process-rules'
+LISTS = 'list'  # singular, as it is the prefix of the file
```

Comment:
Wouldn't List as name be better?
List = 'list'
or did I miss something here (as it represents a single List object, not Lists object)